Classes/Controller/Backend/ControlCenter/TemplaVoilaPlus8UpdateController.php

```diff
@@ -441,14 +441,14 @@ protected function checkAllPageContentForTo(array $validatedToWithDs): array
                 if (isset($validatedToWithDs[$row['tx_templavoilaplus_to']])) {
                     $validatedToWithDs[$row['tx_templavoilaplus_to']]['countUsage'] += $row['COUNT(`uid`)'];
                 } else {
-                    $validationErrors[] = 'There are pages which use an non existent Template Object with uid "' . $row['tx_templavoilaplus_to'] . '" like page with page uid: "' . $row['uid'] . '"';
+                    $validationErrors[] = 'There is a page(s) using a non-existent Template Object with uid "' . $row['tx_templavoilaplus_to'] . '" like page with page uid: "' . $row['uid'] . '"';
```

Comment:
“There is at least one page” should be used here

Reply:
Good catch... I have updated this in 7496824.